### ISSUE TITLE
Getting error with v1.2.0

### ISSUE THREAD

**Issue by layflags** [rse/node-inline-assets#3]
call: `inline-assets --htmlmin --cssmin --pattern 'css$' --verbose some.html`

error:
```
[...]/node_modules/inline-assets/lib/inline-assets.js:205
            if (!attr.src.match(/^(?:data|https?):/)) {
                         ^

TypeError: Cannot read property 'match' of undefined
    at [...]/node_modules/inline-assets/lib/inline-assets.js:205:26
    at RegExp.[Symbol.replace] (native)
    at String.replace (native)
    at inlineAssetsForHTML ([...]/node_modules/inline-assets/lib/inline-assets.js:200:23)
    at inlineAssets ([...]/node_modules/inline-assets/lib/inline-assets.js:290:16)
    at Object.<anonymous> ([...]/node_modules/inline-assets/bin/inline-assets.js:106:11)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[...]/node_modules/inline-assets/lib/inline-assets.js:205
            if (!attr.src.match(/^(?:data|https?):/)) {
                         ^
```

was working with v1.1.1

**Comment by rse** [comment]
Should be now fixed with version 1.2.1

**Comment by layflags** [comment]
Awesome, thx! 👍
